Add build instructions to the ReadMe (#30)
DataDog/fluent-plugin-datadog: README.md

  type kubernetes_metadata
</filter>
```

## Build

To build a new version of this plugin and push it to RubyGems:

- Update the version in the .gemspec file accordingly
- `rake build` to build the gem file
- `rake release` to push the new gem to RubyGems

> Note: the latest command will fail if you don't have the appropriate credentials, one way to set them is to run the following command:

Reviewer: ```suggestion
**Note**: The latest command will fail without appropriate credentials configured. You can set those credentials by running the following command:
```
